UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -320,9 +319,12 @@ public virtual void RemoveFromBodyThis()
 
 
 		/// <summary>
-		/// Checks if a body part has a condition or state that applies logic upon removal then executes it.
+		/// Checks if it's possible to remove this body part and runs any logic
+		/// required upon it's removal.
 		/// </summary>
-		private void BodyPartRemovalChecks()
+		/// <returns>True if allowed to remove. Flase if gibbing.</returns>
+		[ContextMenu("Debug - Drop this Body Part")]
+		private bool BodyPartRemovalChecks()
```

Comment:
I think this has to be public to work with context menu. Did you test it works being private?

Reply:
misplaced context menu, whoops.

Reply:
wait a minute, this thing isn't misplaced on my end. what le fok github?

Reply:
fixed it by editing the file directly from github itself.